src/main/rest-api/IdGenerator.js

```diff
@@ -0,0 +1,14 @@
+export class IdGenerator {
+    constructor (idLength = 6, charaters = 'abcdefghijklmnopqrstuvwxyz') {
+        this.idLength = idLength;
+        this.charaters = charaters;
+    }
+
+    generateId() {
+        let id = '';
+        for (let indexOfCharacter = 1; indexOfCharacter <= this.idLength; indexOfCharacter++) {
+            id += this.charaters.charAt(Math.round(Math.random() * this.charaters.length));
```

Comment:
i don't know if maybe we can use not a random and try to vinculate with the name of the scenario.